### PR TITLE
Ltfoa small fixes

### PR DESCRIPTION
**Pull request by AFoletti**
fixes
- https://github.com/geoadmin/mf-chsdi3/issues/1390
- https://github.com/geoadmin/mf-geoadmin3/issues/2277
- https://github.com/geoadmin/mf-geoadmin3/issues/2260

### CODE REVIEW

File: chsdi/templates/htmlpopup/temperaturmessnetz.mako [hunk 1]
```diff
@@ -34,9 +43,9 @@
 % endif
         </td>
    </tr>
+   <tr><td>${_('date_time')}</td>   <td>${html_attr['date_time']}</td></tr>
```
Comment (gjn) [inline chsdi/templates/htmlpopup/temperaturmessnetz.mako]: This fails if html_attr is None

File: chsdi/templates/htmlpopup/temperaturmessnetz.mako [hunk 2]
```diff
@@ -34,9 +43,9 @@
...
    <tr><td class="cell-left">${_('ch.bafu.hydrologie-wassertemperaturmessstationen.name')}</td>    <td>${c['attributes']['name'] or '-'}</td></tr>
    <tr><td>${_('ch.bafu.hydrologie-wassertemperaturmessstationen.id')}</td>   <td>${c['featureId'] or '-'()}</td></tr>
-   <tr><td>${_('date_time')}</td>   <td>${html_attr['date_time']}</td></tr>
```
Comment (gjn) [inline chsdi/templates/htmlpopup/temperaturmessnetz.mako]: Dito

File: chsdi/templates/htmlpopup/temperaturmessnetz.mako [hunk 3]
```diff
@@ -34,9 +43,9 @@
...
    <tr><td>${_('wassertemperatur')}</td>    <td>${html_attr['wassertemperatur']}</td></tr>
```
Comment (gjn) [inline chsdi/templates/htmlpopup/temperaturmessnetz.mako]: Dito

**Comment by AFoletti** [comment]
shoud be OK now :)

**Comment by loicgasser** [comment]
test jenkins

**Comment by loicgasser** [comment]
jenkins test

**Comment by gjn** [comment]
Thanks alot.

**Pull request merged**